grails-app/i18n/messages.properties

```
@@ -1073,6 +1074,29 @@ fulfillmentItem.available.label=Available
 fulfillmentItem.packed.label=Packed
 fulfillmentItem.picked.label=Picked
 fulfillmentItem.requested.label=Requested
+
+# Messages related to outbound import
+fulfillmentRequest.origin.invalid=Origin can't be the same as destination
+fulfillmentRequest.destination.invalid=Destination can't be the same as origin
+fulfillmentRequest.destination.nullable=Destination cannot be null
```

```suggestion
fulfillmentRequest.destination.nullable=Destination cannot be empty
```
I feel like a non-developer would understand "empty" more than "null". Same comment for all these.